README.md

```diff
@@ -137,12 +141,93 @@ Compilation
 
 To compile with Visual C++ 2019, open the directory and let CMake do its thing.
 If you have the Windows DDK installed correctly, it should just work.
+Note : you need to use the Ninja generator for CMake, else the driver might not work once installed
 
 To compile with GCC on Linux, you will need a cross-compiler set up, for either
 `i686-w64-mingw32` or `x86_64-w64-mingw32`. Create a build directory, then use
 either `mingw-x86.cmake` or `mingw-amd64.cmake` as CMake toolchain files to
 generate your Makefile.
 
+Signing
+-----------
+To install a built-from-source or a nightly (from github actions) driver, you need to sign it
+In this section, I will show how to disable driver signature enforcement, or how to sign the driver using test-signing
+Warning : both of these solutions aren't perfect, if you are looking for a everyday-use it is recommanded to use the release builds, which are signed with a Microsoft-trusted key.
+
+
+## Disabling signature enforcement
+Note : this isn't persistent accross reboots
+
+Relevant link : https://docs.microsoft.com/en-us/windows-hardware/drivers/install/installing-an-unsigned-driver-during-development-and-test
+
+### 1 - Reboot your computer without driver signature enforcement
+
+(This can be done by pressing the 'Shift' key while rebooting the computer, this will take you to a special screen from which you can select "Troubleshoot" -> "Startup settings" -> "Disable driver signature enforcement")
+
+### 2 - Install the driver
+
+Open the previously downloaded artifact from Github Actions (or mimic the directory structure and fill it with the binaries you just compiled), right-click the btrfs.inf file -> install
+
+A window saying "Microsoft can't verify the publisher of this driver" should pop-up. Click "Install anyway"
+
+If the window didn't show up, the driver probably wasn't installed, make sure you booted your computer with driver signature enforcement disabled.
+
+The computer should next tell you to reboot your computer
+
+### 3 - Reboot the computer again without driver signature enforcement
+Follow the same instructions as step 1
+
+### 4 - Done !
+The driver should now be installed ! Note it will only load when the computer is started without driver signature enforcement
```

Comment:
```suggestion
### 4 - Done!

The driver should now be installed ! Note it will only load when the computer is started without driver signature enforcement
```